.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
Please remove your text editor config from PR.

Reply:
Actually this is an editor-agnostic config, this is why i've added it. I thought that it's more welcoming for contributors tom have it because their editor will automatically pick it up and configure accordingly.

Reply:
Ok let’s keep it then